files/jquery.serialscroll/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "bower",

```

Comment:
I'm seeing it show up on http://bower.io/search/?q=jquery.serialScroll and it installs fine for me with `bower install jquery.serialScroll`.  Am I missing something?